app/editor/yarn.lock

```diff
@@ -15258,9 +15258,9 @@ __metadata:
   languageName: node
   linkType: hard
 
-"tno-core@file:./tno-core-0.1.134.tgz::locator=mmi-editor-app%40workspace%3A.":
-  version: 0.1.134
-  resolution: "tno-core@file:./tno-core-0.1.134.tgz::locator=mmi-editor-app%40workspace%3A."
+"tno-core@file:./tno-core-0.1.136.tgz::locator=mmi-editor-app%40workspace%3A.":
+  version: 0.1.136
+  resolution: "tno-core@file:./tno-core-0.1.136.tgz::locator=mmi-editor-app%40workspace%3A."
```

Comment:
You didn't correctly run `yarn install` to fix this lock.  You manually changed it.

Reply:
The last push is executing `yarn install`.